src/main/java/com/happeningnow/service/ServiceComment.java

```diff
@@ -0,0 +1,17 @@
+package com.happeningnow.service;
+
+import com.happeningnow.model.Comment;
+import com.happeningnow.repository.CommentRepository;
+import com.happeningnow.service.factory.ServiceFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ServiceComment extends ServiceFactory<Comment, CommentRepository> {
+
+    private final CommentRepository commentRepository;
```

Comment:
this variable is not being used, you may remove it.

Reply:
Ok